study.md

```diff
@@ -160,15 +171,36 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+After receiving and interpreting a request message, a web server responds with an HTTP response message. The HTTP response should specify the content type of the body. This is done in the header, in the Content-Type field; for instance: Content/Type: application/json
+
+Here are some common status codes and their meaning:
+  200 OK
+  This response code indicates that the request was successful.
+
+  201 Created
+  This indicates the request was successful and a resource was created. It is used to confirm success of a PUT or POST request.
+
+  400 Bad Request
+  The request was malformed. This happens especially with POST and PUT requests, when the data does not pass validation, or is in the wrong format.
+
+  404 Not Found
+  This response indicates that the required resource could not be found. This is generally returned to all requests which point to a URL with no corresponding resource.
+
+  401 Unauthorized
+  This error indicates that you need to perform authentication before accessing the resource.
+
+  405 Method Not Allowed
+  The HTTP method used is not supported for this resource.
+
+
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl -v reddit.com
```

Comment:
Not sure what this command is I'm not familiar with it. This is the more common one:

`curl --request GET https://www.reddit.com`